src/index.html

```diff
@@ -7,6 +7,9 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Lato');
```

Comment:
No se debe importar aquí la fuente.

Reply:
Ya lo noté, de hecho ni siquiera la lee así :v (subo una versión sin eso)